terraform/docs/tagging-template.md

```diff
@@ -0,0 +1,80 @@
+# Terraform Tagging Template
+
+This template provides the standard tagging configuration for all terraform deployments.
+
+## Standard AWS Provider Configuration
+
+```hcl
+provider "aws" {
+  region = "eu-west-1"
+  default_tags {
+    tags = local.default_tags
+  }
+}
+
+locals {
+  default_tags = {
+    Product              = "GOV.UK"
+    System               = "[SYSTEM_NAME]"  # Replace with appropriate system name
+    Environment          = var.govuk_environment
+    Owner                = "govuk-platform-engineering@digital.cabinet-office.gov.uk"
+    repository           = "govuk-infrastructure"
+    terraform_deployment = basename(abspath(path.root))
+  }
+}
+```
+
+## Standard Google Provider Configuration
+
+```hcl
+provider "google" {
+  project = var.google_project_id
+  region  = "europe-west2"
+  default_labels = {
+    product              = "govuk"
+    system               = "[system-name]"  # Replace with lowercase system name
+    environment          = var.govuk_environment
+    owner                = "govuk-platform-engineering"
+    repository           = "govuk-infrastructure"
+    terraform_deployment = basename(abspath(path.root))
+  }
+}
+```
+
+## Resource-Specific Tagging
+
+For resources that need additional tags:
+
+```hcl
+resource "aws_example_resource" "example" {
+  # ... resource configuration ...
+  
+  tags = merge(local.default_tags, {
```

Comment:
Default tags set on the provider are set on all resources created by that provider. They do not need to be explicitly specified